BrainPortal/lib/tasks/seed_test_bourreau.rake

```diff
@@ -0,0 +1,37 @@
+
+#
+# CBRAIN Project
+#
+# Copyright (C) 2008-2012
+# The Royal Institution for the Advancement of Learning
+# McGill University
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+namespace :db do
+  namespace :seed do
+    namespace :test do
+      desc "Seed CBRAIN test DB for Bourreau testing"
+      task :bourreau => :environment do
+        raise "Error: this task must be run in a TEST environment!" unless
```

Comment:
I think you can add the common command `rake db:seed:test:bourreau RAILS_ENV=test` just to be more clear.